01_standalone_examples/docker-compose-with-postgres/README.md

```diff
@@ -0,0 +1,4 @@
+# lakeFS Docker Compose with PostgreSQL
+
+This is an example Docker Compose file showing how to use lakeFS with PostgreSQL as the K/V store and MinIO as the S3-compatible Object Store. 
```

Comment:
I lean towards removing any mention of "kv store" in our docs (except "under the hood" type things)...
```suggestion
This is an example Docker Compose file showing how to use lakeFS with PostgreSQL as the database and MinIO as the underlying storage.
```